cypress.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I think we ended overriding the `defaultTimeout` in both elements & preview, might as well here too